frontend/app/src/components/accounts/blockchain/Eth2Input.vue

```diff
@@ -92,10 +92,23 @@ watch(
 </script>
 
 <template>
-  <div class="grid gap-4 grid-cols-2">
-    <div class="col-span-2">
+  <div class="grid gap-4 grid-cols-3 mt-3">
+    <div class="col-span-3 md:col-span-1">
+      <AmountInput
```

Comment:
Do you think we can also replace the `AmountInput` with a normal `RuiTextInput`. 

Validator index is an identifier that happens to be a number, we don't show it as an amount which is correct, but we add it as an amount which we should not.

Reply:
updated